test/e2e/deployment.go

```diff
@@ -149,6 +152,23 @@ func checkDeploymentRevision(c *clientset.Clientset, ns, deploymentName, revisio
 	return deployment, newRC
 }
 
+func stopDeployment(c *clientset.Clientset, oldC client.Interface, ns, deploymentName string) {
```

Comment:
oldC will be removed soon in #20452. f.Client will return *clientset.Clientset. cc @caesarxuchao